### PR TITLE
[Feature][UI-Next]: Build a front-end refactoring project.

### PR DESCRIPTION
**Pull request by songjianet**
## Purpose of the pull request

Build a front-end refactoring project.
close [#7380](https://github.com/apache/dolphinscheduler/issues/7380)

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

### CODE REVIEW

File: dolphinscheduler-ui-next/.eslintignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+node_modules
```
Comment (wangyizhi1) [inline dolphinscheduler-ui-next/.eslintignore:16]: Missing license header.

Comment (songjianet) [reply dolphinscheduler-ui-next/.eslintignore:16]: Okay, I will deal with this issue later

**Comment by lenboo** [comment]
What does this pr do? 
Can you give an accurate message in title and content?

**Comment by songjianet** [comment]
> What does this pr do? Can you give an accurate message in title and content?

sure

File: dolphinscheduler-ui-next/.eslintignore (hunk 1 above)
Comment (songjianet) [reply dolphinscheduler-ui-next/.eslintignore:16]: complete

**Comment by songjianet** [comment]
> What does this pr do? Can you give an accurate message in title and content?

complete

**Review by wangyizhi1 — APPROVED** [review]
LGTM

**Pull request closed without merging**